deploy/kubernetes/Readme.md

```diff
@@ -0,0 +1,35 @@
+# Deploy Tracee on Kubernetes
+
+> NOTE `Minimal Requirements to run tracee in the node`
+````
+https://aquasecurity.github.io/tracee/install/prerequisites/
+````
```

Comment:
Can you add a note that 

> This approach assumes that kernel headers are available on the Kubernetes nodes under conventional location, e.g `/usr/src` and `/lib/modules`

Reply:
after merging this we should spend some time improving this experience: instructions to build the eBPF probe once using the tracee image, and then use the slim image in the daemonset. for a first step this is fine